### ISSUE TITLE
Some questions and feature ideas

### ISSUE THREAD

**Issue by cptvincer** [efck-chat-keyboard/efck#5]
Fantastic, more features then i knew i wanted- got it for emojis and theres so much more...

Question: is there a way to keep the window open or minimize/maximize? The app start a tad slower then i often need in a pinch, theres not many aliases (finding an emoji by similar words) and many new emojis i wouldnt even knew how to find but scrolling... on selecting an emoji to apply the window auto-closes- and after that is another slow startup and the list back at the top

And if not possible those are my main suggestions:
-options for not closing, always on top
-being able to minimize to taskbar

**Comment by kernc** [comment]
Thanks for opening the first issue!

May I ask on what platform/hardware are you finding the app slow to start?

You can view all aliases in a tooltip if you hover over an emoji, and you can assign [custom emoji aliases](https://efck-chat-keyboard.github.io/faq/#emoji-custom-strings) (please don't trip over the wild joke 🙏).

-----

I think I understand your two feature requests perfectly as both already came up in beta:
* **Use an intermediate buffer to be able to paste several emojis in sequence.**
  Or, alternatively, a paste-and-stay-open behavior, but this may be harder to achieve reliably on all platforms due to all the async messaging to/from the window manager. In either case, accompanied by a related keybind, such as <kbd>Ctrl</kbd>+<kbd>Enter</kbd>.
* **Use a background app/daemon/systray agent to alleviate slow startup.**
  In my testing so far, the app was ready in under <500ms (measured with `--debug`) on both Linux and macOS on some decent old hardware. So I figured, why keep a daemon if a popup can do. Slow startup on Windows I attributed to Microsoft and VirtualBox.

**Comment by cptvincer** [comment]
Yup im o windows 10. Ryzen 5 3600, 16 ram. Might be something on how microsoft handles things.
Thats why i sugested some form of 'minimize'- it wouldnt eliminate the delay, only shift things so it happens only once (first startup in a session) instead of every time

**Comment by Symbian9** [comment]
> being able to minimize to taskbar

Minimizing app to taskbar (instead of closing app) would be good.

**Comment by kernc** [comment]
There is PR https://github.com/efck-chat-keyboard/efck/pull/11 if any of you want to help/test it. It works for me on X11 but currently seems to crash on Windows. You can download [built artifacts here on the checks page](https://github.com/efck-chat-keyboard/efck/actions/runs/3791186441#artifacts).

**Comment by Symbian9** [comment]
> You can download [built artifacts here on the checks page](https://github.com/efck-chat-keyboard/efck/actions/runs/3791186441#artifacts).

I just tried to install DEB - there is issue that nightly build has a lower version than last release:

> Error: A later version is already installed.

If remove stable — it installs, but there is another issue:

- https://github.com/efck-chat-keyboard/efck/issues/14
